dist/profile/templates/kubernetes/resources/evergreen_gateway/deployment.yaml.erb

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: gateway
+  namespace: evergreen
+spec:
+  replicas: 3
```

Comment:
It's not clear to me why this Deployment requires _3_ replicas? Please add some commentary to the YAML explaining a bit more please.

Reply:
It's only for redundancy purpose.

Reply:
It doesn't really matter on which gateway we arrive since they don't store any state.